staff/acct/sorry/sorry

```diff
@@ -54,7 +54,7 @@ else
     export KRB5CCNAME
 fi
 
-sorryshell=/opt/ocf/bin/sorried
+sorryshell=/opt/share/utils/bin/sorried
```

Comment:
I've done it here and in puppet; probably should go over every repo just to be sure. I almost forgot about LDAP.